common/test/acceptance/pages/lms/login.py

```diff
@@ -16,37 +16,41 @@ class LoginPage(PageObject):
     url = BASE_URL + "/login"
 
     def is_browser_on_page(self):
-        return any([
-            'log in' in title.lower()
-            for title in self.q(css='span.title-super').text
-        ])
+        return self.browser.title.startswith("Sign in or Register")
 
-    def login(self, email, password):
+    def login(self, email, password, expected_redirect=None):
         """
         Attempt to log in using `email` and `password`.
+        `expected_redirect` is a PageObject that corresponds to the page
+        that we expect the user to be redirected to after login.
         """
         self.provide_info(email, password)
-        self.submit()
+        self.submit(expected_redirect)
 
     def provide_info(self, email, password):
         """
         Fill in login info.
         `email` and `password` are the user's credentials.
         """
-        EmptyPromise(self.q(css='input#email').is_present, "Click ready").fulfill()
-        EmptyPromise(self.q(css='input#password').is_present, "Click ready").fulfill()
+        EmptyPromise(self.q(css='input#login-email').is_present, "Click ready").fulfill()
```

Comment:
I think so actually. The only instance of `LoginPage` in LMS tests just checks to see that it loads, nothing else. I'm guessing every other test used `AutoAuth`. So now we have a simple test that logging in works!